llm_ollama.py

```diff
@@ -235,3 +271,26 @@ def _get_ollama_models() -> List[dict]:
         return ollama.list()["models"]
     except:
         return []
+
+
+def _ollama_model_capability_completion(model: str) -> bool:
+    """Check if a model is capable of completion.
+    This is a indicator for if a model can be used for chat or if its an embedding only model.
```

Comment:
This chat capability check is fragile. I understand there is no other "official" way, so this is the best we can do. Still, I'd ask you to extend the docstring to include an explanation of where this implementation comes from (with permalinks to Ollama code.) In the event that changes in Ollama break this check, such explanation will help the future people to unbreak this code.